x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_custom_assets_extension.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { useKibana } from '../../../../../common/lib/kibana';
+import { APP_PATH } from '../../../../../../common/constants';
+import {
+  CustomAssetsAccordionProps,
+  CustomAssetsAccordion,
+  PackageAssetsComponent,
+} from '../../../../../../../fleet/public';
+
+export const EndpointCustomAssetsExtension: PackageAssetsComponent = () => {
+  const { http } = useKibana().services;
+  const views: CustomAssetsAccordionProps['views'] = [
+    {
+      name: 'Hosts',
+      url: http.basePath.prepend(`${APP_PATH}/hosts/allHosts`),
```

Comment:
it makes more sense for this to go to: `${APP_PATH}/administration/endpoints` - this is where users can perform Admin tasks on Endpoint that they deploy through Fleet.

Unless you have Endpoints deployed, you'll see an Onboarding screen.  But if you do have some deployed, you'd see them like this

![image](https://user-images.githubusercontent.com/56395104/124026749-0eca0180-d9c0-11eb-93f8-06f3e002882e.png)